main.go

```diff
@@ -1,8 +1,8 @@
 package main
 
 import (
-	"github.com/alexellis/k3sup/pkg/cmd"
 	"github.com/spf13/cobra"
+	"github.com/utsavanand2/k3sup/pkg/cmd"
```

Comment:
This is wrong and shouldn't change. 

Clone or move the repo to $HOME/go/src/github.com/alexellis/k3sup

Then run `export GOPATH=$HOME/go`

Reply:
I was confused about this too! Thanks!